prac_05/hex_colours.py

```diff
@@ -0,0 +1,9 @@
+HEX_COLOURS = {"AliceBlue": "#f0f8ff", "aquamarine1": "#7fffd4"}
+
+colour = input("Enter name of colour: ")
```

Comment:
How could you improve this code to handle a user entering "aliceblue" instead of "AliceBlue".  You could use .casefold()
it returns AliceBlue as aliceblue.  Give it a try in the console, it's pretty cool.